exercises/macros/tests/macros.rs

```diff
@@ -67,6 +67,16 @@ fn test_nested() {
     );
 }
 
+mod test {
+    use macros::hashmap;
+    #[test]
+    #[ignore]
+    fn type_not_in_scope() {
+        let _empty = hashmap!();
```

Comment:
You need a type annotation for the compiler to be able to work with this.

```suggestion
        let _empty: ::std::collections::HashMap<(), ()> = hashmap!();
```

Reply:
Hmm. I thought I had tested that. I guess I didn't.